.github/workflows/pr.yaml

```
@@ -41,6 +43,40 @@ jobs:
     uses: rapidsai/shared-workflows/.github/workflows/checks.yaml@branch-24.10
     with:
       enable_check_generated_files: false
+  fetch-pandas-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      pandas-versions: ${{ steps.get-versions.outputs.versions }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Use latest major versions of actions.
```suggestion
        uses: actions/checkout@v4
```